.editorconfig

```diff
@@ -0,0 +1,40 @@
+﻿root=true
+
+[*]
+end_of_line = crlf
```

Comment:
on Windows crlf is fine, it is what is default in most editors. But it is recommended to have crlf = auto in the git config.